### DISCUSSION TITLE
Is there a way to listen for when the scanning is canceled?

### DISCUSSION THREAD

**Discussion by Lukydemboy** [capawesome-team/capacitor-mlkit#132] [General]
I want to run some functionality when the scanning is canceled (when the close button is clicked). However I can't seem to find a method to do this, maybe this would be great to do with the addEventListener?

**Comment by robingenz** [comment]
An error is thrown if the user cancels the scan, see https://github.com/capawesome-team/capacitor-mlkit/blob/3f55a650148a1fcdc2873abd9a632272a75824d3/packages/barcode-scanning/android/src/main/java/io/capawesome/capacitorjs/plugins/mlkit/barcodescanning/BarcodeScannerPlugin.java#L206
